api/views.py

```diff
@@ -45,6 +47,15 @@ class ExportInstructorLocationsView(ListAPIView):
     serializer_class = ExportInstructorLocationsSerializer
 
 
+class ExportMembersView(ListAPIView):
+    """Show everyone who qualifies as an SCF member."""
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    paginator = None  # disable pagination
+
+    queryset = get_members()
```

Comment:
Does the framework know to call `get_queryset`, or will we have to do 
that explicitly somewhere? And is it a regular method or a class method?